course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -248,6 +268,73 @@
             $q.all([peoplePromise, memberPromise])
                 .then($scope.handleLookupResults);
         };
+        $scope.removeMembership = function(membership) {
+            // the call stack to get here is a little weird.  we register
+            // this as the success callback on a promise hung off the
+            // $uibModal instance.
+            var courseMemberURL = djangoUrl.reverse(
+                                      'icommons_rest_api_proxy',
+                                      ['api/course/v2/course_instances/' +
+                                       $scope.courseInstanceId +
+                                       '/people/' + membership.user_id]);
+            var config = {
+                data: {
+                    role_id: membership.role.role_id,
+                    user_id: membership.user_id,
+                },
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            };
+            $http.delete(courseMemberURL, config)
+                .success(function(data, status, headers, config, statusText) {
+                    var success = membership; // TODO - copy to avoid stomping the original?
+                    success.searchTerm = membership.profile.name_last +
+                                         ', ' + membership.profile.name_first;
+                    success.action = 'removed from';
+                    $scope.successes.push(success);
+                    $scope.dtInstance.reloadData()
+                })
+                .error(function(data, status, headers, config, statusText) {
+                    $scope.handleAjaxError(data, status, headers, config, statusText);
+
+                    var failure = membership; // TODO - copy to avoid stomping the original?
+                    var reloadData = false; // for some partial failures we need to reload
+                    switch(status) {
+                        case 404:
+                            switch (data.detail) {
+                                case 'User not found.':
+                                    failure.type = 'noSuchUser';
+                                    break;
```

Comment:
Seems like we should probably refresh the datatable here as well so that person disappears from the visible results.